packages/terra-alert/src/terra-dev-site/doc/example/ActionAndDismissibleExample.jsx

```diff
@@ -40,26 +38,13 @@ class ActionAndDismissibleExample extends React.Component {
     }
     return (
       <div>
-        <div dir="ltr">
-          <h3>Alert of type custom with no icon or title that is dismissible and has an action button</h3>
-          <br />
-          <code>
-            {alertStartTagStr}
-            <br />
-            &nbsp;&nbsp;
-            {alertText}
-            <br />
-            {alertEndTagStr}
-          </code>
-        </div>
-        <br />
         {alertElem}
         <br />
         <p>
-Action button has been clicked
+          Action button has been clicked
```

Comment:
There is a missing a space between clicked and the clickCount

Reply:
Updated: https://github.com/cerner/terra-core/pull/2130/commits/b6cc3d82c58603e2146e1eb06400d6dd8dbc51ca